test/devices_list_test.rb

```diff
@@ -234,4 +235,56 @@ def partition_name
       end
     end
   end
+
+  describe Y2Storage::DevicesLists::LvmVgsList do
+    let(:scenario) { "lvm-two-vgs" }
+    let(:vgs) { fake_devicegraph.vgs }
+
+    it "contains all volume groups by default" do
+      expect(vgs.size).to eq 2
+      expect(described_class.new(fake_devicegraph).size).to eq 2
```

Comment:
To be honest I see this `described_class.new(fake_devicegraph)` too many times in test. What about defining `subject { described_class.new(fake_devicegraph) }` ?